daemon/container_operations_windows.go

```diff
@@ -3,8 +3,16 @@
 package daemon
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/cloudflare/cfssl/log"
```

Comment:
This is copied from `container_operations_unix.go` and I prefer to keep the files in sync initially.